tests/SprykerSdkTest/Spryk/Integration/Glue/AddGlueBasicStructureTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Integration\Glue;
+
+use Codeception\Test\Unit;
+use SprykerSdkTest\Module\ClassName;
+
+/**
+ * Auto-generated group annotations
+ * @group SprykerSdkTest
+ * @group Spryk
+ * @group Integration
+ * @group Glue
+ * @group AddGlueBasicStructureTest
+ * Add your own group annotations below this line
+ */
+class AddGlueBasicStructureTest extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\SprykIntegrationTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testAddsGlueConfigConstant(): void
+    {
+        $this->tester->run($this, [
+            '--module' => 'FooBar',
+            '--resourceType' => 'foo-bars',
+            '--fromTransfer' => 'FooBarTransfer',
```

Comment:
Maybe it's better to explicitly set it to code mode? `'--mode' => 'core',`

Reply:
Idk, it's done like this every other tests, I don't think, that this is really needed.